package-lock.json

```diff
@@ -217,6 +211,13 @@
       "integrity": "sha512-FZdkNBDqBRHKQ2MEbSC17xnPFOhZxeJ2YGSfr2BKf3sujG49Qe3bB+rGCwQfIaA7WHnGeGkSijX4FuBCdrzW/g==",
       "dev": true
     },
+    "abbrev": {
```

Comment:
Why are there additions if you're only upgrading dependencies?

Reply:
@berickson1 I suppose it is due to some of NPM internal changes., Some packages depend on `abbrev`, for instance `fsevents`, `nopt`. Need to check how NPM builds dependencies tree.